### PR TITLE
Fix version

### PR DESCRIPTION
**Pull request by jburel**
Changes in the recent version of ilastik broke the notebook and the script
This PR fixes the issue and caps the version so we do not have a surprise before meeting

Check that you can run the notebooks and the python scripts
cc @dominikl @pwalczysko

### CODE REVIEW

File: notebooks/pixel_classification.ipynb [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "from getpass import getpass\n",
     "from collections import OrderedDict\n",
     "\n",
-    "import ilastik_main\n",
+    "from ilastik import main\n",
```
Comment (mtbc) [inline notebooks/pixel_classification.ipynb]:
```suggestion
    "from ilastik import app\n",
```
Don't really know, just wondering!

Comment (jburel) [reply notebooks/pixel_classification.ipynb]: well spotted

**Comment by pwalczysko** [comment]
Is this a successful run ?

```
(notebook) pwalczysko@6e9beb5102f8:~/scripts$ python pixel_classification_zarr.py
Host [wss://outreach.openmicroscopy.org/omero-ws]:
Username [trainer-1]:
Password:
Dataset ID [6161]:
INFO ilastik.app: Starting ilastik from "/srv/conda/envs/notebook/ilastik-meta".
INFO:ilastik.app:Starting ilastik from "/srv/conda/envs/notebook/ilastik-meta".
Starting ilastik from "/srv/conda/envs/notebook/ilastik-meta".
INFO ilastik.app: Resetting lazyflow thread pool with 2 threads.
INFO:ilastik.app:Resetting lazyflow thread pool with 2 threads.
INFO ilastik.app: Configuring lazyflow RAM limit to 2.0GiB
INFO:ilastik.app:Configuring lazyflow RAM limit to 2.0GiB
INFO lazyflow.utility.memory: Available memory set to 2.0GiB
INFO:lazyflow.utility.memory:Available memory set to 2.0GiB
WARNING 2020-05-15 14:11:56,057 memory 120 140410733053760 User specified memory exceeds memory physically available. Please check theconfiguration.
WARNING:lazyflow.utility.memory:User specified memory exceeds memory physically available. Please check theconfiguration.
WARNING 2020-05-15 14:11:57,085 __init__ 120 140410733053760 Failed to automatic tracking workflow; check dependencies: libemon.so.1.3.1: cannot open shared object file: No such file or directory
WARNING:ilastik.workflows.tracking.conservation:Failed to automatic tracking workflow; check dependencies: libemon.so.1.3.1: cannot open shared object file: No such file or directory
WARNING 2020-05-15 14:11:57,088 __init__ 120 140410733053760 Failed to import automatic tracking workflow (conservation tracking). For this workflow, see the installationinstructions on our website ilastik.org; check dependencies: libemon.so.1.3.1: cannot open shared object file: No such file or directory
WARNING:ilastik.workflows:Failed to import automatic tracking workflow (conservation tracking). For this workflow, see theinstallationinstructions on our website ilastik.org; check dependencies: libemon.so.1.3.1: cannot open shared object file:No such file or directory
WARNING 2020-05-15 14:11:57,103 __init__ 120 140410733053760 Failed to import structured learning tracking workflow. For this workflow, see the installationinstructions on our website ilastik.org; check dependencies: libemon.so.1.3.1: cannot open shared object file: No such file or directory
WARNING:ilastik.workflows:Failed to import structured learning tracking workflow. For this workflow, see the installationinstructions on our website ilastik.org; check dependencies: libemon.so.1.3.1: cannot open shared object file: No such file or directory
INFO ilastik.shell.projectManager: Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
INFO:ilastik.shell.projectManager:Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
running ilastik using ../notebooks/pipelines/pixel-class-133.ilp and B3.tif
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
INFO ilastik.applets.batchProcessing.batchProcessingApplet: Exporting to in-memory array.
INFO:ilastik.applets.batchProcessing.batchProcessingApplet:Exporting to in-memory array.
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO:lazyflow.utility.bigRequestStreamer:Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO lazyflow.utility.bigRequestStreamer: determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO:lazyflow.utility.bigRequestStreamer:determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO lazyflow.utility.bigRequestStreamer: Chose blockshape: (1, 92, 92, 92, 2)
INFO:lazyflow.utility.bigRequestStreamer:Chose blockshape: (1, 92, 92, 92, 2)
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per block is 748.6MiB
INFO:lazyflow.utility.bigRequestStreamer:Estimated RAM usage per block is 748.6MiB
WARNING arraytypes.py(1263): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1269): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
DEBUG lazyflow.operators.classifierOperators: Features took 7.6188 seconds. Prediction took 1.676011 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
DEBUG:lazyflow.operators.classifierOperators:Features took 7.6188 seconds. Prediction took 1.676011 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
Killed
(notebook) pwalczysko@6e9beb5102f8:~/scripts$
```

**Comment by joshmoore** [comment]
`Killed` seems worrisome.

**Comment by jburel** [comment]
You run out of memory. Ilastik started so the change has been applied

**Comment by pwalczysko** [comment]
The two notebooks (zarr and zarr idr) work fine.

The two scripts (pixel_class and pixel_class zarr) work fine too

**Comment by jburel** [comment]
I forgot to mention that the "parallel" one will not work when using repo2docker or binder

**Comment by pwalczysko** [comment]
When trying to run the "pixel_class" notebook (the only one without the word zarr in it), after connection, there is no "True" output. Can you add it there if possible ? I actually made me to conclude that the connection did not succeeed. In the other notebooks, the feedback "True" is there

**Comment by pwalczysko** [comment]
Ilastik definitely starts also when the third notebook (the one with no zarr) is run

**Comment by jburel** [comment]
done

**Comment by pwalczysko** [comment]
See output of the parallel script run locally, I think it worked too.

<details>


```
(test-2) pwalczysko@ls31618~/Work/omero-guide-ilastik/scripts$ python pixel_classification_zarr_parallel.py 
Host [wss://outreach.openmicroscopy.org/omero-ws]: 
Username [trainer-1]: 
Password: 
Dataset ID [6161]: 
INFO ilastik.app: Starting ilastik from "/Users/pwalczysko/miniconda3/envs/test-2/ilastik-meta".
INFO ilastik.app: Starting ilastik from "/Users/pwalczysko/miniconda3/envs/test-2/ilastik-meta".
Starting ilastik from "/Users/pwalczysko/miniconda3/envs/test-2/ilastik-meta".
Starting ilastik from "/Users/pwalczysko/miniconda3/envs/test-2/ilastik-meta".
INFO ilastik.app: Resetting lazyflow thread pool with 2 threads.
INFO ilastik.app: Resetting lazyflow thread pool with 2 threads.
INFO ilastik.app: Configuring lazyflow RAM limit to 2.0GiB
INFO ilastik.app: Configuring lazyflow RAM limit to 2.0GiB
INFO lazyflow.utility.memory: Available memory set to 2.0GiB
INFO lazyflow.utility.memory: Available memory set to 2.0GiB
WARNING 2020-05-15 16:00:14,660 opConservationTracking 20858 123145461018624 Could not find any ILP solver
WARNING 2020-05-15 16:00:14,660 opConservationTracking 20860 123145478152192 Could not find any ILP solver
WARNING 2020-05-15 16:00:14,672 opStructuredTracking 20860 123145478152192 Could not find any ILP solver
WARNING 2020-05-15 16:00:14,672 opStructuredTracking 20858 123145461018624 Could not find any ILP solver
WARNING 2020-05-15 16:00:14,673 structuredTrackingWorkflow 20860 123145478152192 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
WARNING 2020-05-15 16:00:14,673 structuredTrackingWorkflow 20858 123145461018624 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
INFO ilastik.shell.projectManager: Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
INFO ilastik.shell.projectManager: Opening Project: ../notebooks/pipelines/pixel-class-133.ilp
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSimpleBlockedArrayCache.py(43): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
WARNING opSlicedBlockedArrayCache.py(141): FutureWarning: Conversion of the second argument of issubdtype from `dtype` to `np.generic` is deprecated. In future, it will be treated as `np.object_ == np.dtype(dtype).type`.
INFO ilastik.applets.batchProcessing.batchProcessingApplet: Exporting to in-memory array.
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO lazyflow.utility.bigRequestStreamer: determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO lazyflow.utility.bigRequestStreamer: Chose blockshape: (1, 92, 92, 92, 2)
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per block is 748.6MiB
WARNING arraytypes.py(1263): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1269): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
INFO ilastik.applets.batchProcessing.batchProcessingApplet: Exporting to in-memory array.
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per pixel is 504.0B * safety factor (2.0)
INFO lazyflow.utility.bigRequestStreamer: determining blockshape assuming available_ram is 1.5GiB, split between 2 threads
INFO lazyflow.utility.bigRequestStreamer: Chose blockshape: (1, 92, 92, 92, 2)
INFO lazyflow.utility.bigRequestStreamer: Estimated RAM usage per block is 748.6MiB
WARNING arraytypes.py(1263): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
WARNING arraytypes.py(1269): FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
DEBUG lazyflow.operators.classifierOperators: Features took 5.487577 seconds. Prediction took 2.708923 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.556577 seconds. Prediction took 2.689021 seconds. Subregion: start '[0, 0, 0, 0]' stop '[92, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 8.953349 seconds. Prediction took 3.601168 seconds. Subregion: start '[0, 0, 92, 0]' stop '[92, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 9.080312 seconds. Prediction took 3.688201 seconds. Subregion: start '[0, 0, 92, 0]' stop '[92, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.094743 seconds. Prediction took 2.77286 seconds. Subregion: start '[0, 0, 184, 0]' stop '[92, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.761243 seconds. Prediction took 3.088859 seconds. Subregion: start '[0, 0, 184, 0]' stop '[92, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.998273 seconds. Prediction took 2.222957 seconds. Subregion: start '[0, 0, 276, 0]' stop '[92, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.360741 seconds. Prediction took 2.75842 seconds. Subregion: start '[0, 92, 0, 0]' stop '[92, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.136672 seconds. Prediction took 3.215258 seconds. Subregion: start '[0, 92, 0, 0]' stop '[92, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.909623 seconds. Prediction took 3.208362 seconds. Subregion: start '[0, 92, 92, 0]' stop '[92, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.353783 seconds. Prediction took 4.44951 seconds. Subregion: start '[0, 92, 92, 0]' stop '[92, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.292308 seconds. Prediction took 3.585777 seconds. Subregion: start '[0, 92, 184, 0]' stop '[92, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.68287 seconds. Prediction took 2.838202 seconds. Subregion: start '[0, 92, 184, 0]' stop '[92, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.518761 seconds. Prediction took 2.6003540000000003 seconds. Subregion: start '[0, 184, 0, 0]' stop '[92, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.3041409999999996 seconds. Prediction took 2.076305 seconds. Subregion: start '[0, 92, 276, 0]' stop '[92, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.102768 seconds. Prediction took 2.929385 seconds. Subregion: start '[0, 184, 92, 0]' stop '[92, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.06279 seconds. Prediction took 2.741587 seconds. Subregion: start '[0, 184, 0, 0]' stop '[92, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.261299 seconds. Prediction took 2.5060029999999998 seconds. Subregion: start '[0, 184, 184, 0]' stop '[92, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.643048 seconds. Prediction took 2.69496 seconds. Subregion: start '[0, 184, 92, 0]' stop '[92, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.437076 seconds. Prediction took 2.79723 seconds. Subregion: start '[92, 0, 0, 0]' stop '[184, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.932693 seconds. Prediction took 2.810716 seconds. Subregion: start '[0, 184, 184, 0]' stop '[92, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.225573 seconds. Prediction took 1.390112 seconds. Subregion: start '[0, 184, 276, 0]' stop '[92, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.33872 seconds. Prediction took 0.451473 seconds. Subregion: start '[0, 276, 0, 0]' stop '[92, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.051137 seconds. Prediction took 0.151071 seconds. Subregion: start '[0, 276, 92, 0]' stop '[92, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.119064 seconds. Prediction took 0.218018 seconds. Subregion: start '[0, 276, 184, 0]' stop '[92, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.89959 seconds. Prediction took 3.070709 seconds. Subregion: start '[92, 0, 92, 0]' stop '[184, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.64855 seconds. Prediction took 0.139092 seconds. Subregion: start '[0, 276, 276, 0]' stop '[92, 281, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.488284 seconds. Prediction took 4.325735 seconds. Subregion: start '[92, 0, 184, 0]' stop '[184, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.084218 seconds. Prediction took 3.090171 seconds. Subregion: start '[92, 0, 0, 0]' stop '[184, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 8.022706 seconds. Prediction took 3.695232 seconds. Subregion: start '[92, 92, 0, 0]' stop '[184, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 8.583981 seconds. Prediction took 3.7807180000000002 seconds. Subregion: start '[92, 0, 92, 0]' stop '[184, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.769224 seconds. Prediction took 3.333296 seconds. Subregion: start '[92, 92, 92, 0]' stop '[184, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.7701080000000005 seconds. Prediction took 3.007031 seconds. Subregion: start '[92, 0, 184, 0]' stop '[184, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.567791 seconds. Prediction took 2.651526 seconds. Subregion: start '[92, 0, 276, 0]' stop '[184, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.959292 seconds. Prediction took 3.284235 seconds. Subregion: start '[92, 92, 184, 0]' stop '[184, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.288875 seconds. Prediction took 3.026626 seconds. Subregion: start '[92, 184, 0, 0]' stop '[184, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.295411 seconds. Prediction took 3.205595 seconds. Subregion: start '[92, 92, 0, 0]' stop '[184, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.3200520000000004 seconds. Prediction took 3.116202 seconds. Subregion: start '[92, 184, 92, 0]' stop '[184, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 8.087034 seconds. Prediction took 3.762504 seconds. Subregion: start '[92, 92, 92, 0]' stop '[184, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.372374 seconds. Prediction took 3.537964 seconds. Subregion: start '[92, 184, 184, 0]' stop '[184, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 8.205961 seconds. Prediction took 3.654489 seconds. Subregion: start '[92, 92, 184, 0]' stop '[184, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.061199 seconds. Prediction took 1.9472930000000002 seconds. Subregion: start '[184, 0, 0, 0]' stop '[236, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.227384 seconds. Prediction took 2.456441 seconds. Subregion: start '[92, 92, 276, 0]' stop '[184, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.778385 seconds. Prediction took 1.62854 seconds. Subregion: start '[184, 0, 92, 0]' stop '[236, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.72112 seconds. Prediction took 1.839729 seconds. Subregion: start '[184, 0, 184, 0]' stop '[236, 92, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.477881 seconds. Prediction took 3.12486 seconds. Subregion: start '[92, 184, 0, 0]' stop '[184, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.080888 seconds. Prediction took 1.8335240000000002 seconds. Subregion: start '[184, 92, 0, 0]' stop '[236, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 4.4480509999999995 seconds. Prediction took 1.720936 seconds. Subregion: start '[184, 92, 92, 0]' stop '[236, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 7.472566 seconds. Prediction took 3.56327 seconds. Subregion: start '[92, 184, 92, 0]' stop '[184, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.952727 seconds. Prediction took 1.983865 seconds. Subregion: start '[184, 92, 184, 0]' stop '[236, 184, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 6.650141 seconds. Prediction took 2.867657 seconds. Subregion: start '[92, 184, 184, 0]' stop '[184, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.400914 seconds. Prediction took 1.859594 seconds. Subregion: start '[184, 184, 0, 0]' stop '[236, 275, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.634909 seconds. Prediction took 0.517998 seconds. Subregion: start '[92, 276, 0, 0]' stop '[184, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 5.239176 seconds. Prediction took 1.388462 seconds. Subregion: start '[92, 184, 276, 0]' stop '[184, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.43766 seconds. Prediction took 1.52925 seconds. Subregion: start '[184, 184, 92, 0]' stop '[236, 275, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.932461 seconds. Prediction took 0.149438 seconds. Subregion: start '[92, 276, 92, 0]' stop '[184, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 1.028719 seconds. Prediction took 0.219884 seconds. Subregion: start '[92, 276, 184, 0]' stop '[184, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.660477 seconds. Prediction took 0.117044 seconds. Subregion: start '[92, 276, 276, 0]' stop '[184, 281, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.171445 seconds. Prediction took 1.4202029999999999 seconds. Subregion: start '[184, 184, 184, 0]' stop '[236, 275, 271, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.357613 seconds. Prediction took 1.3365179999999999 seconds. Subregion: start '[184, 0, 0, 0]' stop '[239, 92, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.72155 seconds. Prediction took 1.5971419999999998 seconds. Subregion: start '[184, 0, 92, 0]' stop '[239, 92, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.8595829999999998 seconds. Prediction took 1.292483 seconds. Subregion: start '[184, 0, 184, 0]' stop '[239, 92, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.379933 seconds. Prediction took 1.044252 seconds. Subregion: start '[184, 0, 276, 0]' stop '[239, 92, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.581617 seconds. Prediction took 1.288629 seconds. Subregion: start '[184, 92, 0, 0]' stop '[239, 184, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.28132 seconds. Prediction took 1.5373109999999999 seconds. Subregion: start '[184, 92, 92, 0]' stop '[239, 184, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.044263 seconds. Prediction took 1.433607 seconds. Subregion: start '[184, 92, 184, 0]' stop '[239, 184, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.572939 seconds. Prediction took 1.022672 seconds. Subregion: start '[184, 92, 276, 0]' stop '[239, 184, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.646605 seconds. Prediction took 1.356623 seconds. Subregion: start '[184, 184, 0, 0]' stop '[239, 276, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 3.36752 seconds. Prediction took 1.555222 seconds. Subregion: start '[184, 184, 92, 0]' stop '[239, 276, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.930519 seconds. Prediction took 1.301874 seconds. Subregion: start '[184, 184, 184, 0]' stop '[239, 276, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.546185 seconds. Prediction took 0.24006 seconds. Subregion: start '[184, 276, 0, 0]' stop '[239, 281, 92, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 2.173984 seconds. Prediction took 0.763715 seconds. Subregion: start '[184, 184, 276, 0]' stop '[239, 276, 325, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.429808 seconds. Prediction took 0.066211 seconds. Subregion: start '[184, 276, 92, 0]' stop '[239, 281, 184, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.517311 seconds. Prediction took 0.102334 seconds. Subregion: start '[184, 276, 184, 0]' stop '[239, 281, 276, 2]'
DEBUG lazyflow.operators.classifierOperators: Features took 0.286288 seconds. Prediction took 0.037036 seconds. Subregion: start '[184, 276, 276, 0]' stop '[239, 281, 325, 2]'
Compute time (in minutes): 3.0
done
/Users/pwalczysko/miniconda3/envs/test-2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/test-2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/test-2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
/Users/pwalczysko/miniconda3/envs/test-2/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))
(test-2) pwalczysko@ls31618~/Work/omero-guide-ilastik/scripts$ 
```

</details>

Edit Yes, it did work, as I can see and open the two pngs saved locally.

**Comment by pwalczysko** [comment]
this looks good https://github.com/ome/omero-guide-ilastik/pull/15/commits/dcc7b404303a0ac8e5cfb9a97d582077c90edc20

Ready to merge fmpov

**Comment by jburel** [comment]
Thanks @pwalczysko for checking

**Pull request merged**